build.gradle

```diff
@@ -0,0 +1,67 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java project to get you started.
+ * For more details take a look at the Java Quickstart chapter in the Gradle
+ * User Manual available at https://docs.gradle.org/6.6.1/userguide/tutorial_java_projects.html
+ */
```

Comment:
I really like that there are description on specific lines of the gradle to explain the use of that particular line